src/OrbitQt/main.cpp

```diff
@@ -342,6 +387,15 @@ int main(int argc, char* argv[]) {
     return -1;
   }
 
-  RunUiInstance(deployment_configuration, &context.value());
+  // If more than one capture files are provided, start multiple Orbit instances.
+  QString orbit_executable = QString::fromStdString(argv[0]);
```

Comment:
`argv[0]` is not an `std::string`, but `char*`, so it would be better to use the constructor `QString(argv[0])`. 
personal preference: I would move this line up. Probably to the very top, but definitely before `QApplication app(argc, argv);`, just in case that QApplication constructor changes `argv`

Reply:
Changed accordingly. Thanks so much for pointing out this :-) !